superset/assets/webpack.config.js

```diff
@@ -12,15 +12,15 @@ const VERSION_STRING = JSON.parse(fs.readFileSync('package.json')).version;
 
 const config = {
   entry: {
-    'css-theme': APP_DIR + '/javascripts/css-theme.js',
-    common: APP_DIR + '/javascripts/common.js',
+    // 'css-theme': APP_DIR + '/javascripts/css-theme.js',
+    // common: APP_DIR + '/javascripts/common.js',
     dashboard: ['babel-polyfill', APP_DIR + '/javascripts/dashboard/Dashboard.jsx'],
     explore: ['babel-polyfill', APP_DIR + '/javascripts/explore/explore.jsx'],
     explorev2: ['babel-polyfill', APP_DIR + '/javascripts/explorev2/index.jsx'],
-    sqllab: ['babel-polyfill', APP_DIR + '/javascripts/SqlLab/index.jsx'],
-    standalone: ['babel-polyfill', APP_DIR + '/javascripts/standalone.js'],
-    welcome: ['babel-polyfill', APP_DIR + '/javascripts/welcome.js'],
-    profile: ['babel-polyfill', APP_DIR + '/javascripts/profile/index.jsx'],
+    // sqllab: ['babel-polyfill', APP_DIR + '/javascripts/SqlLab/index.jsx'],
```

Comment:
Oh this is my "optimization" hack to get faster build times while iterating, I'll have to remove before merging for sure!